src/components/apps/launch/AppInfo.js

```diff
@@ -77,13 +95,19 @@ const AppInfo = ({
                         app?.name
                     )}
                 </Typography>
-            </Toolbar>
-
+                <Hidden xsDown>
+                    <Typography
+                        className={classes.appInfoTypography}
+                        variant="body2"
+                        display="block"
+                        gutterBottom
+                        noWrap={true}
+                    >
+                        {loading ? <Skeleton /> : app?.description}
+                    </Typography>
+                </Hidden>
+            </Paper>
             <Box m={2}>
```

Comment:
Replaced `Toolbar` with `Paper` so that the launch screen surface looks uniform (white bg).

Reply:
Actually the `Paper` here was excess and not required. So I removed it. But I needed an element to set `ref` to calculated `AppInfo` bar height dynamically. So added a `div`.